core/src/components/input/input.tsx

```diff
@@ -343,6 +347,10 @@ export class Input implements ComponentInterface {
       ...inheritAriaAttributes(this.el),
       ...inheritAttributes(this.el, ['tabindex', 'title', 'data-form-type']),
     };
+
+    if (this.labelPlacement === undefined) {
+      this.labelPlacement = getIonTheme(this) === 'ionic' ? ionicThemeDefaultLabelPlacement : 'start';
```

Comment:
Since the `ionicThemeDefaultLabelPlacement` variable was created to track the value, should another variable to created to track the value `start` for the other themes?

Reply:
I didn't do this because the `start` default value is only used once, right here. The ionic theme default is also used in the runtime check and printed warning up in `getLabelPlacement()`.